src/js_types.hpp

```diff
@@ -24,6 +24,7 @@
 #include <string>
 #include <vector>
 #include <sstream>
+#include <iostream>
```

Comment:
Is this include required, or a leftover from debugging?

Reply:
yep :), good catch 👍